services/management/dbaas/xtradb_cluster_service.go

```diff
@@ -287,6 +287,32 @@ func (s XtraDBClusterService) RestartXtraDBCluster(ctx context.Context, req *dba
 	return &dbaasv1beta1.RestartXtraDBClusterResponse{}, nil
 }
 
+// GetXtraDBClusterResources returns expected resources to be consumed by the cluster.
+func (s XtraDBClusterService) GetXtraDBClusterResources(ctx context.Context, req *dbaasv1beta1.GetXtraDBClusterResourcesRequest) (*dbaasv1beta1.GetXtraDBClusterResourcesResponse, error) {
+	settings, err := models.GetSettings(s.db.Querier)
+	if err != nil {
+		return nil, err
+	}
+
+	clusterSize := int64(req.Params.ClusterSize)
+	memory := (req.Params.Pxc.ComputeResources.MemoryBytes + req.Params.Proxysql.ComputeResources.MemoryBytes) * clusterSize
+	cpu := int64(req.Params.Pxc.ComputeResources.CpuM+req.Params.Proxysql.ComputeResources.CpuM) * clusterSize
+	disk := (req.Params.Pxc.DiskSize + req.Params.Proxysql.DiskSize) * clusterSize
+
+	if settings.PMMPublicAddress != "" {
+		memory += 1000000000 * clusterSize
```

Comment:
@tiagomotasantos I don't understand why is it 1G? 
`kubectl describe pod newclustername-pxc-2`
```
Containers:
  pmm-client:
    Container ID:   docker://512e26e1b24c8eed9f33df532c8fc1e1710d694c658d87255af045c84678684d
    Image:          perconalab/pmm-client:dev-latest
    Image ID:       docker-pullable://perconalab/pmm-client@sha256:e6b04f556b0bf9702b2d9164122b9d6416245c7fe245669f0baa31221402302c
    Ports:          7777/TCP, 30100/TCP, 30101/TCP, 30102/TCP, 30103/TCP, 30104/TCP, 30105/TCP
    Host Ports:     0/TCP, 0/TCP, 0/TCP, 0/TCP, 0/TCP, 0/TCP, 0/TCP
    State:          Running
      Started:      Tue, 09 Mar 2021 11:49:28 +0100
    Last State:     Terminated
      Reason:       Error
      Exit Code:    137
      Started:      Tue, 09 Mar 2021 08:59:23 +0100
      Finished:     Tue, 09 Mar 2021 11:12:26 +0100
    Ready:          True
    Restart Count:  2
    Requests:
      cpu:     500m
      memory:  500M
```